packages/adapter-netlify/index.js

```diff
@@ -51,7 +51,7 @@ module.exports = function () {
 			copyFileSync(resolve(__dirname, 'files/index.cjs'), `${functions}/render/index.js`);
 
 			// create _redirects
-			writeFileSync(`${publish}/_redirects`, '/* /.netlify/functions/render 200');
+			appendFileSync(`${publish}/_redirects`, '/* /.netlify/functions/render 200');
```

Comment:
`appendFileSync` will not insert a newline
https://replit.com/@samccone/KindheartedRosybrownBookmark

The docs for netlify say that rules must be separated by a newline
https://docs.netlify.com/routing/redirects/#syntax-for-the-redirects-file

For this case we will want to handle

1. the no file case
2. the empty file but existing case
3. existing file newline terminated case
4. existing file null terminated case

Reply:
Is it possible to add a test for these various cases?

Reply:
Yes, but ...😄  I've done more research on Netlify redirects.

In "[Rule processing order](https://docs.netlify.com/routing/redirects/#rule-processing-order)", they state the following:

> Rules in the _redirects file are always processed first, followed by rules in the Netlify configuration file.

My understanding is any `[[redirect]]` rules defined in `netlify.toml` ([docs](https://docs.netlify.com/configure-builds/file-based-configuration/#redirects)) will be ignored if we add `/*` to `_redirects` (regardless of whether we create or append to the file).

Given we already ask users of the `netlify-adapter` to create a `netlify.toml` file, what are your thoughts on asking users to add the following redirect - at the end of the existing redirects if they already have some configured:

```
[[redirects]]
  from = "/*"
  to = "/.netlify/functions/render"
  status = 200
```

Alternatively, which also assists with a migration path for existing users of the adapter, we could check if `netlify.toml` contains the redirect and if not, add it programmatically.

Reply:
I already pointed out the `_redirects` over `netlify.toml` priority in my initial post. That's why I suggested to describe it in the adapter's readme. But you're right. We should try to make it a little smarter. https://github.com/sveltejs/kit/issues/930#issuecomment-817077432 should cover everything. What do you think?

btw, @mikenikles did you try out the fix locally? Did you get the same "index.js should be named index.cjs" error?

Reply:
Ah my bad, apologies for repeating what you already said. Yep, I think your comment (https://github.com/sveltejs/kit/issues/930#issuecomment-817077432) covers the possible situations.

I saw the `index.cjs` error once when I ran `netlify dev` in the generated `build` directory. I realized that's the wrong place and when I ran `netlify dev` in the directory with the `netlify.toml` file, I had no issues.